app/main.py

```diff
@@ -16,4 +16,8 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, collection_of_coins, marks],
```

Comment:
The `collection_of_coins` is a set, which is an immutable type in Python. It should be categorized under 'immutable' instead of 'mutable'.